setup.py

```diff
@@ -156,7 +156,7 @@ def write_version(filename=os.path.join(*['airflow',
     'hmsclient>=0.1.0',
     'pyhive>=0.1.3',
     'impyla>=0.13.3',
-    'thrift_sasl==0.2.1',
+    'thrift_sasl==0.2.1'
```

Comment:
I strongly believe we should move to `thrift_sasl` version `0.3.0`. This is the one that they are using as well: https://github.com/dropbox/PyHive/blob/master/dev_requirements.txt#L17

Reply:
Could you remove this dependency? This is pulled in by the PyHive dependency recursively.